openpype/hosts/unreal/api/plugin.py

```diff
@@ -1,7 +1,251 @@
+                    # we have legacy instance
+                    family = i.get("family")
+                    if (family not in
+                            shared_data["unreal_cached_legacy_subsets"]):
+                        shared_data[
+                            "unreal_cached_legacy_subsets"][family] = [i]
+                    else:
+                        shared_data[
+                            "unreal_cached_legacy_subsets"][family].append(i)
+                    continue
+
+                creator_id = i.get("creator_identifier")
+                if creator_id not in shared_data["unreal_cached_subsets"]:
+                    shared_data["unreal_cached_subsets"][creator_id] = [i]
+                else:
+                    shared_data["unreal_cached_subsets"][creator_id].append(i)
+        return shared_data
+
+    def create(self, subset_name, instance_data, pre_create_data):
+        try:
+            instance_name = f"{subset_name}{self.suffix}"
+            pub_instance = create_publish_instance(instance_name, self.root)
+
+            instance_data["subset"] = subset_name
+            instance_data["instance_path"] = f"{self.root}/{instance_name}"
+
+            instance = CreatedInstance(
+                self.family,
+                subset_name,
+                instance_data,
+                self)
+            self._add_instance_to_context(instance)
+
+            pub_instance.set_editor_property('add_external_assets', True)
+            assets = pub_instance.get_editor_property('asset_data_external')
+
+            ar = unreal.AssetRegistryHelpers.get_asset_registry()
+
+            for member in pre_create_data.get("members", []):
+                obj = ar.get_asset_by_object_path(member).get_asset()
+                assets.add(obj)
+
+            imprint(f"{self.root}/{instance_name}", instance_data)
+
+        except Exception as er:
+            six.reraise(
+                OpenPypeCreatorError,
+                OpenPypeCreatorError(f"Creator error: {er}"),
+                sys.exc_info()[2])
+
+    def collect_instances(self):
+        # cache instances if missing
+        self.cache_subsets(self.collection_shared_data)
+        for instance in self.collection_shared_data[
+                "unreal_cached_subsets"].get(self.identifier, []):
+            created_instance = CreatedInstance.from_existing(instance, self)
+            self._add_instance_to_context(created_instance)
+
+    def update_instances(self, update_list):
+        unreal.log_warning(f"Update instances: {update_list}")
+        for created_inst, _changes in update_list:
+            instance_node = created_inst.get("instance_path", "")
+
+            if not instance_node:
+                unreal.log_warning(
+                    f"Instance node not found for {created_inst}")
+
+            new_values = {
+                key: new_value
+                for key, (_old_value, new_value) in _changes.items()
+            }
```

Comment:
Changes object has changed
```suggestion
        for created_inst, changes in update_list:
            instance_node = created_inst.get("instance_path", "")

            if not instance_node:
                unreal.log_warning(
                    f"Instance node not found for {created_inst}")

            new_values = {
                key: changes[key].new_value
                for key in changes.changed_keys()
            }
```